.github/ISSUE_TEMPLATE/icon_request.yml

```
@@ -0,0 +1,48 @@
+name: Icon request
+description: Request a new icon for Simple Icons
+labels: [new icon]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        We won't add non-brand icons or anything related to illegal services. If in
+        doubt, open an issue and we'll have a look. Before opening a new issue, please
+        search for duplicate or closed issues or PRs. If you find one for the brand
+        you're requesting then leave a comment on it or add a reaction to the original
+        post.
+
+  - type: input
+    attributes:
+      label: Brand Name
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Website
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Alexa Rank
+      description: >-
+        The Alexa rank can be retrieved at https://www.alexa.com/siteinfo/. Please see
+        our contributing guidelines for more details on how we assess a brand's
+        popularity.
```

Not sure, but if this is parsed as markdown I think we should link to the Contributing Guidelines, e.g.:

```suggestion
      description: >-
        The Alexa rank can be retrieved at https://www.alexa.com/siteinfo/. Please see
        our [contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) for more details on how we assess a brand's
        popularity.
```

Else we could go with something similar to what the existing template does:

```suggestion
      description: >-
        The Alexa rank can be retrieved at https://www.alexa.com/siteinfo/. Please see
        our contributing guidelines for more details on how we assess a brand's
        popularity, which can be found at https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md.
```

---

It does support markdown, but the link was moved to the top of the form.